server/utils/string.js

```diff
@@ -0,0 +1,20 @@
+const { capitalize } = require('./index');
+
+const fullNameOr = (placeHolder, ...names) => {
+  const fullName = names.map(capitalize).join(' ').trim();
+  return fullName === '' ? placeHolder : fullName;
+};
+
+const formatBalanceOr = (placeHolder, amount, currency) => {
+  return amount && currency
+    ? new Intl.NumberFormat('en-GB', {
+        style: 'currency',
+        currency,
+      }).format(amount)
+    : placeHolder;
+};
+
+module.exports = {
+  fullNameOr,
```

Comment:
Not gonna not LGTM this but I've never been a fan of function names ending half way through a sentence as they aren't self descriptive, as in Or what? They only make sense if you know what the args are. Same with For something as well, you're passing something in so it has to be For those args anyway.

Reply:
Renamed to `..OrDefault` to be more explicit with intent